tlf/id.go

```diff
@@ -227,6 +231,21 @@ func (id *ID) UnmarshalText(buf []byte) error {
 	return id.UnmarshalBinary(bytes)
 }
 
+// MarshalJSON implements the json.Marshaler interface for ID.
+func (id ID) MarshalJSON() ([]byte, error) {
+	return keybase1.Quote(id.String()), nil
+}
```

Comment:
As an alternative where we don't use Quote, I think implementing encoding.TextMarshaler (which would just return `id.String()`) would also work.

Reply:
I was being consistent with the KID mashaling. But I don't really follow your point.  Does the golang json encoder call the `TextMarshaler` interface if its available?  I don't see that mentioned in the docs.

Reply:
Yeah it's mentioned in the `encoding/json` doc: 

> If no MarshalJSON method is present but the value implements encoding.TextMarshaler instead, Marshal calls its MarshalText method and encodes the result as a JSON string. 

It might have been introduced later than the KID stuff was written. I think this seems cleaner but wouldn't cry if you choose to keep Quote here!

Reply:
Oh I see, I don't know how I missed that when I searched the documentation, thanks.  Actually `tlf.ID` already has `MarshalText`, so I guess I'll just remove this part of the change entirely.